### DISCUSSION TITLE
Did something break the cybis build?

### DISCUSSION THREAD

**Discussion by KC5VDJ** [kej715/DtCyber#73] [General]
5.7.43, nos 2.8.7, debug and release, mickeysoft backend and Intel 19.2 backend, default optimizations on both, optimized Intel actually going the farthest.

All bomb building cybis, usually around the modmove of the cybis user from 1 to 377777, except the last (optimized Intel), which got this far...

This is after I think six attempts at nos 2.8.7 install full, win 10, latest, vs22, latest, Intel classic 19.2, Lenovo laptop, haswell (avx2), node, and related latest as of this past weekend.

![8P4iIVzY7D.png](https://github.com/kej715/DtCyber/assets/136194499/da9f1b3a-4ea2-4308-a752-36d5bf217c01)

**Comment by KC5VDJ** [comment]
Might be ether-crud....  this time it is now building the courseware....  no changes on my end, I did it because I know this compiler and settings built it before....  why does cybis etc have to be downloaded if present, and if online, when it is not changed?  Same for reconfigure.  There needs to be an offline cache option that when online only downloads what is changed, and if ether-crud messing with things, better crc or even ecc to assure a sound copy.

Never had a need to get into JavaScript/ECMAScript,looks like I might need to... 😁 It is a bit of an improvement in ways over tk/tcl...but the pattern matching of the former...

In fact I do want to edit these tape mount timeouts as well, unless good for file substitutions at the last second.  It isn't like we have to wait for an operator to free him/herself to get and load a tape.

**Comment by KC5VDJ** [comment]
Figures .  Walk away and...host not found...

She will sing...

![43VvhSkzyH.png](https://github.com/kej715/DtCyber/assets/136194499/dfa34685-5867-4706-ac20-287aff24aeaf)

![cmd_991vaywBce.png](https://github.com/kej715/DtCyber/assets/136194499/063778c9-cef9-47c6-93c1-e99f86e26b40)

**Reply by KC5VDJ** [reply]
![Fat.Lady.Singing.jpg.jpg](https://github.com/kej715/DtCyber/assets/136194499/4d4c74b0-7434-4933-a356-c25bcaa98b38)

I cheated after another connection error.... Straight node install.  But she is singing...

I'll have to dig into the JavaScript at some point.

![cmd_OS2ELtcYbU.png](https://github.com/kej715/DtCyber/assets/136194499/07f7c433-0cde-47cb-aaba-7543571283f5)

![12OTjJ6hIL.png](https://github.com/kej715/DtCyber/assets/136194499/a4f70289-35fe-452d-a83f-9753b71deed6)

**Comment by kej715** [comment]
Please elaborate. If a simple "node install" failed, please provide some
details about how it failed. For example, what error message(s) were issued.
